sdk/src/main/java/com/hedera/hashgraph/sdk/HederaTrustManager.java

```diff
@@ -108,7 +108,7 @@ public void checkServerTrusted(X509Certificate[] chain, String authType) throws
                 throw new IllegalStateException("Failed to find SHA-384 digest for certificate hashing", e);
             }
 
-            if (this.certHash.equals(Hex.toHexString(certHashBytes))) {
+            if (this.certHash.equalsIgnoreCase(Hex.toHexString(certHashBytes))) {
```

Comment:
The hashes should be identical, I don't see the point in comparing them case-insensitive.

Reply:
Hexadecimal is not case sensitive.

Reply:
I see, makes sense. Though I am still cautious about changing this part of the logic, because
1. It's quite old and I do not have the whole context for it
2. It seems to be involving the security of the sdk
3. I checked this piece of logic in other sdks and there it's the same as the current implementation